### PR TITLE
Update language in form without using the cookie

### PR DESCRIPTION
**Pull request by Steph4104**
Issue #89 

Not using the cookie anymore. Put the toggle in a form.
It should keep the information of the user in a session. Need to be test in preprod.

### CODE REVIEW

File: core/templates/language.html [hunk 1]
```diff
@@ -18,44 +18,46 @@
...
-                <div class="menu" title="{% trans "Change language" %}">
-                  
-                  {% for langs in LANGUAGES %}
-                  <a href="{{ current_path }}" class="item" onclick="changeLanguage('{{ langs.0 }}');">   
-                      {{ langs.0|language_name_local }}
-                  </a>
-                  {% endfor %}
+        <form name="ui" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+            <input name="next" type="hidden" value="{{ redirect_to }}" />
+            <input name="language" type="hidden"/>
```
[the remark this answers is not in the document]
Comment (bryan-robitaille) [reply core/templates/language.html]: added hidden label to input to abide by w3c standards

File: core/templates/language.html [hunk 2]
```diff
@@ -18,44 +18,46 @@
...
-              <i class="dropdown icon"></i>
-                <div class="menu" title="{% trans "Change language" %}">
-                  
-                  {% for langs in LANGUAGES %}
-                  <a href="{{ current_path }}" class="item" onclick="changeLanguage('{{ langs.0 }}');">   
-                      {{ langs.0|language_name_local }}
-                  </a>
-                  {% endfor %}
+        <form name="ui" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+            <input name="next" type="hidden" value="{{ redirect_to }}" />
```
[the remark this answers is not in the document]
Comment (bryan-robitaille) [reply core/templates/language.html]: added hidden label to hidden input field to abide by w3c standards

**Comment by Steph4104** [comment]
@bryan-robitaille , I update the toggle and put it in a form. I keep the changeLanguage function so it should create the cookie.

The dropdown appear only if in your settings.py you have more than 2 language (in our case, French, english and dutch ) Otherwise, you should only see one bottom.

**Comment by bryan-robitaille** [comment]
Tested and good to go

**Comment by bryan-robitaille** [comment]
Stupid me... wrong button.  Closed by accident

**Pull request merged**